### PR TITLE
Implement currently tracked courses UI

### PR DESCRIPTION
**Pull request by haiyingweng**
## Overview

Implement currently tracked courses UI with some dummy data

## Changes Made

- made `TrackedCourseCard` for each tracked course/section
- added tracked courses to `TrackedCoursesView`

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>

  
Screen Shot 2021-04-09 at 6 17 27 PM

</details>

### CODE REVIEW

File: src/components/TrackedCourses/TrackedCoursesView.tsx [hunk 1]
```diff
@@ -2,15 +2,90 @@ import React from 'react'
...
+      }
+    }
     return (
       <div className="tracked-courses-view">
-        <svg height="70" width="70">
-          <circle cx="35" cy="35" r="35" fill="#47C753" />
-        </svg>
-        <p className="tracking-title-label">No Courses Currently Tracked</p>
-        <p className="tracking-subtitle-label">Tap the search icon to start adding courses</p>
+        {sections()}
```
Comment (mathewscullin) [inline src/components/TrackedCourses/TrackedCoursesView.tsx:88]: In what case is this return for? Doesn't the else above catch-all?

**Review by mathewscullin — APPROVED** [review]
Wonderful work! 🙌

File: src/components/TrackedCourses/TrackedCourseCard.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          section.status === Status.OPEN
+            ? <button className="enroll-button">ENROLL</button>
+            : null
+        }
+      </div>
+    </div >
+  )
+}
+
+
```
Comment (YKo20010) [inline src/components/TrackedCourses/TrackedCourseCard.tsx:61]: delete newlines

File: src/components/TrackedCourses/TrackedCoursesView.tsx (hunk 1 above)
Comment (YKo20010) [reply src/components/TrackedCourses/TrackedCoursesView.tsx:88]: looks like sections is a function so it's just being called here

File: src/components/TrackedCourses/TrackedCourseCard.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import { Section, Status } from '../../types'
+
+export interface TrackedCourseCardProps {
+  section: Section
+}
+
+const TrackedCourseCard: React.FunctionComponent<TrackedCourseCardProps> = ({
+  section
+}) => {
+
```
Comment (YKo20010) [inline src/components/TrackedCourses/TrackedCourseCard.tsx:14]: delete newline

File: src/components/TrackedCourses/TrackedCourseCard.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          </svg>
+        )
+      case Status.WAITLISTED:
+        return (
+          <svg className="status-icon" height="16" width="16">
+            <polygon points="0,16 8,0 16,16" fill="#FFD027" />
+          </svg>
+        )
+    }
+
```
Comment (YKo20010) [inline src/components/TrackedCourses/TrackedCourseCard.tsx:36]: delete newline

**Review by YKo20010 — APPROVED** [review]
lgtm

File: src/components/TrackedCourses/TrackedCoursesView.tsx (hunk 1 above)
Comment (haiyingweng) [reply src/components/TrackedCourses/TrackedCoursesView.tsx:88]: yes yanlam is correct

**Pull request merged**